cypress.config.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+require('dotenv').config({ path: '.env.local' });
```

Comment:
We had this for a hot second when I was setting up the Airtable connection. We deleted it eventually, but we have stuff in our local `.env` files. Are these [docs](https://www.dotenv.org/docs) for `dotenv` a good place where I can figure out why we need it here, but not for Airtable?

Reply:
I would look at at [dotenv's GitHub repo](https://github.com/motdotla/dotenv) rather than those docs. I think that the reason we didn't need it for the Airtable keys was because Next.js auto-magically picks up environment variables, so we didn't need `dotenv`.

Cypress runs totally separate from Next.js, so it doesn't have the built-in functionality to read those variables, so we had to add it in for the `cypress.config.js`.